woocommerce/woocommerce
# Merge REST API into core.

### All Submissions:






## How to review this PR?

This PR contains all commits from `release/1.0` branch from REST API repo so its not practical to review directly. 

I would recommend heading over to https://github.com/vedanshujain/woocommerce/pull/2, <- this PR contains changes that are added on top of merging from `release/1.0` to make sure that API works. Edit: There is also a comment below which documents the merge process to preserve original commit author and details.

I am planning to target 4.5 for this PR, but an early review and merge would be very much appreciated.

### Changes proposed in this Pull Request:



Merge API repo's `release/1.0` into core.

### How to test the changes in this Pull Request:

From https://github.com/vedanshujain/woocommerce/pull/2:

1. Checkout this branch and wc-rest-api plugin (https://github.com/woocommerce/woocommerce-rest-api/). Disable the REST API plugin for now. Run composer update.
2. Go to WooCommerce > Status page and look for WooCommerce REST API package setting string. Make sure that its equal to current WC version.
3. Enable the REST API plugin and refresh the status page. It should now show the API Package version since that;s now loaded instead of API code.

### Other information:




### Changelog entry

> Dev - Merge API Package into core.
